orc8r/cloud/go/services/certifier/storage/storage_blobstore.go

```diff
@@ -30,6 +34,12 @@ const (
 	// CertInfoType is the type of CertInfo used in blobstore type fields.
 	CertInfoType = "certificate_info"
 
+	// UserType is the type of CertInfo used in blobstore type fields.
+	UserType = "user"
+
+	// PolicyType is the type of policy used in blobstore type fileds
+	PolicyType = "policy"
```

Comment:
These types were duplicated earlier in this PR -- let's have just 1 canonical home for them

Reply:
Yeah this is kind of unfortunate. I used it in protos/utils and imported storage + storage imports protos, so duplicated it to break circular dependency, but then again it is also sketchy to put all these types into the protos pkg and use it in storage.

Reply:
Can also potentially create a `const` pkg in certifier/const. I would almost try that. Up to you though